base/Dockerfile

```diff
@@ -1,25 +1,25 @@
 FROM ubuntu:20.04
 
+# Global dependencies
 RUN apt-get -q update \
  && apt-get -q install -y --no-install-recommends apt-utils \
  && apt-get -q install -y --no-install-recommends --allow-downgrades \
  ca-certificates \
  wget \
  libxtst6 \
  libxss1 \
- desktop-file-utils \
  libasound2 \
- libgtk2.0-0 \
+ libgtk-3-0 \
  libnss3 \
- xdg-utils \
  xvfb \
- zenity \
- libdbus-glib-1-2 \
  curl \
  xz-utils \
  cpio \
  lsb-release \
  && apt-get clean
 
+# Support forward compatibility for unity activation
+RUN echo "576562626572264761624c65526f7578" > /etc/machine-id && ln -sf /etc/machine-id /var/lib/dbus/machine-id
```

Comment:
Should we use some kind of Docker ARG with a default value here to allow easier customization?

Reply:
this number should never change, that's the point of it.

Not sure if I understand your meaning.